django/contrib/postgres/fields/hstore.py

```diff
@@ -51,6 +52,19 @@ def formfield(self, **kwargs):
         defaults.update(kwargs)
         return super(HStoreField, self).formfield(**defaults)
 
+    def get_prep_value(self, value):
+        value = super(HStoreField, self).get_prep_value(value)
+
+        if isinstance(value, dict):
```

Comment:
Not sure that will work. `get_prep_value` is used for querying too, so if I remove the test for `dict`, this would raise an error:

    HStoreModel.objects.filter(field__has_keys=['a', 'c'])

In this case, `value` is a list.